stdlib/typing_extensions.pyi

```diff
@@ -59,6 +59,8 @@ from typing import (  # noqa: Y022,Y037,Y038,Y039,UP035
     TypeVar as _TypeVar,
     Union as Union,
     _Alias,
+    _Final as _Final,
+    _SpecialForm as _SpecialForm,
```

Comment:
I think we want these imported from `typing` but do we really want to publicly re-export them from `typing_extensions`? They seem like implementation details of the runtime

```suggestion
    _Final,
    _SpecialForm,
```